app.js

```diff
@@ -33,12 +33,20 @@ const friendData = [
 
```

Comment:
you are never using the `findFriendByName` function, so there is no need to import it. This is preventing github from running your tests and why your build is not passing github actions